tango_simlib/model.py

```diff
@@ -523,7 +525,6 @@ def action_handler(model, data_input=None, tango_dev=None):
                     # Return a default value if output_return is not specified.
                     return_value = ARBITRARY_DATA_TYPE_RETURN_VALUES[action_output_type]
             return return_value
-
```

Comment:
- [ ] I would prefer to keep this line, as it separates the end of the nested `action_handler` function from the body of the outer function.  Having the separation makes it more readable (at least for me).